Libraries/pbxproj/Headers/pbxproj/PBX/Object.h

```diff
@@ -32,7 +32,9 @@ class Object {
 
 protected:
     Object(std::string const &isa);
-
+#if defined(__FreeBSD__) || defined(__OpenBSD__)
+    virtual ~Object(){}
+#endif
```

Comment:
You don't need to gate this with `#if defined(__FreeBSD__) || defined(__OpenBSD__)`. This fix makes sense on all platforms.

Also, you should have a space between `()` and `{}`.